deep_qa/layers/subtract_minimum.py

```diff
@@ -0,0 +1,45 @@
+from keras import backend as K
+
+from deep_qa.layers.masked_layer import MaskedLayer
+
+class SubtractMinimum(MaskedLayer):
+    '''
+    This layer is used to normalize across a tensor axis.  Normalization is done by finding the minimum value
+    across the specified axis, and then subtracting that value from all values (again, across the spcified axis).
```

Comment:
Can you add [input/output/parameter docs](https://github.com/allenai/deep_qa/blob/3a8a98b4fe91ab63e11a801896f65c3e2ae6fdd3/deep_qa/layers/backend/repeat_like.py#L17-L29) to this?  I know this layer is pretty simple, so it's not as necessary, but it's helpful for someone reading the docs to have everything consistent.